db/migrate/20160214115800_inline_ems_id.rb

```diff
@@ -0,0 +1,23 @@
+class InlineEmsId < ActiveRecord::Migration
```

Comment:
This class needs a migration spec as it has a data migration.  When you write the migration spec you will use the `migration_stub` helper method which will make the other comment clearer because it will blow up as is.